### ISSUE TITLE
Build not work

### ISSUE THREAD

**Issue by mrgloom** [gakarak/AirBaseMarking#1]
On Ubuntu 14.04:
~~~
git clone https://github.com/gakarak/AirBaseMarking.git
cd AirBaseMarking
mkdir build && cd build
qmake ..
~~~

Produce error:
~~~
Error processing project file: ..
~~~

**Comment by gakarak** [comment]
Yes, this is my mistake: the ***.pro** file is located in the **src** directory.
you can fix:
 ```
cd AirBaseMarking/src
mkdir build && cd build
qmake ..
```
or
 ```
cd AirBaseMarking
mkdir build && cd build
qmake ../src
```

**Comment by gakarak** [comment]
closed

**Comment by mrgloom** [comment]
`qmake ../src`

~~~
Project ERROR: Unknown module(s) in QT: positioning
~~~

`qmake -version`

~~~
QMake version 3.0
Using Qt version 5.2.1 in /usr/lib/x86_64-linux-gnu
~~~

**Comment by gakarak** [comment]
Thanx, i fix build bug for Qt5 version <5.8 ()
And test on Ubuntu 14.04 with default qt5 installation:
```
$ qmake -version
QMake version 3.0
Using Qt version 5.2.1 in /usr/lib/x86_64-linux-gnu
```
